test/features/test_organized_edge_detection.cpp

```diff
@@ -0,0 +1,126 @@
+/*
```

Comment:
Use SPDX identifier (sample present in `filters/function_filter.h`)

And please use correct copyright.